### DISCUSSION TITLE
Will gqlgen be updated to support stream and defer directive

### DISCUSSION THREAD

**Discussion by bhushanibm** [99designs/gqlgen#2253] [Ideas]
Will stream and defer directive will be supported by gqlgen ?

### versions
 - `go run github.com/99designs/gqlgen version`?
 - `go version`?

**Comment by frederikhors** [comment]
This is an idea. Not an issue.

For the second time I'll move it in discussions.

**Comment by anupriya17** [comment]
I'm looking for support as well. Its definitely helpful for the applications i'm working on

**Comment by fasibio** [comment]
Same dession here: https://github.com/99designs/gqlgen/discussions/1883
Perhaps collect to one ?
